state_daemon:
# note: not all vda 5050 topics are 
	topics_publish:
		state: "/state"
	topics_subscribe:
		header: "/TBD1"				#VDA5050 specific
		orderId: "/TBD2"			#VDA5050 specific
		orderUpdateId: "/TBD3"			#VDA5050 specific
		zoneSetId: "/TBD4"			#VDA5050 specific
		lastNodeId: "/TBD5"   			#VDA5050 specific
		lastNodeSequenceId: "/TBD6"		#VDA5050 specific
	    	nodeState: "/TBD7"			#VDA5050 specific
			# ADD nodeState		
	    	edgeState: "/TBD8"			#VDA5050 specific
			# ADD edgeState
		agvPosition: "/TBD9"			#VDA5050 specific + partially covered by ROS common msg
			positionInitialized: "/TBD10"	#VDA5050 specific
			localizationScore: "/TBD11"	#VDA5050 specific
			deviationRange: "/TBD12"	#VDA5050 specific
			x: "/TBD13"			#ROS msg -> nav_msgs/Odometry Message.pose
			y: "/TBD14"			#ROS msg -> nav_msgs/Odometry Message.pose
			theta: "/TBD15"			#ROS msg -> nav_msgs/Odometry Message.pose
			mapID: "/TBD16"			#VDA5050 specific
			mapDescription: "/TBD17"
		velocity: "/TBD18"
			vx: "/TBD19"			#ROS msg -> nav_msgs/Odometry Message.twist
			vy: "/TBD20"			#ROS msg -> nav_msgs/Odometry Message.twist
			omega: "/TBD21"			#ROS msg -> nav_msgs/Odometry Message.twist
		loads: "/TBD22"
			# ADD loads
		driving: "/TBD23"
		paused: "/TBD10"
		newBaseRequest: "/TBD10"
		distanceSinceLastNode: "/TBD10"
		actionStates: "/TBD10"
			# ADD actionStates
		batteryState: "/TBD10"
			batteryCharge: "/TBD10"
			batteryVoltage: "/TBD10"
			batteryHealth: "/TBD10"
			charging: "/TBD10"
			reach: "/TBD10"
		operatingMode: "/TBD999"
		errors: "/ERROR4389"
			# ADD error
		information:
			# ADD info
		safetyState: "/TBD999"
			eStop: "/TBD999"
			fieldViolation: ""/TBD999"
	
        


topic_error: /error_1

AGV_Data:
    version: 1.1.03
    manufacturer: "template_1"
    serialNumber: "AGV721"




